ansible/iptables.yml

```diff
@@ -0,0 +1,27 @@
+- hosts: kube-node
+  tasks:
+    - name: "Redirect TLS traffic to the kubernetes ingress"
+      become: true
+      iptables:
+        table: "nat"
+        chain: "PREROUTING"
+        in_interface: "eth0"
```

Comment:
This will not work on instances where the interface is `eth0`; can be improved later, probably taking the interface name from the hosts file?